### PR TITLE
Doc port from go sdk and crate readme

### PR DESCRIPTION
**Pull request by supernovahs**

### CODE REVIEW

**Comment by supernovahs** [comment]
Let me know what else should we port from the go sdk

File: crates/services/avsregistry/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Services AvsRegistry
```
Comment (dralves) [inline crates/services/avsregistry/README.md:1]: add example?

File: crates/services/bls_aggregation/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Bls Aggregation Service
+
+## Example
```
Comment (dralves) [inline crates/services/bls_aggregation/README.md:3]: add example?

File: examples/README.md [hunk 3]
```diff
@@ -3,7 +3,19 @@
...
 
 ## OperatorsInfo
-
 ```
 cargo run -p info-operator-service
 ```
+
+## AvsRegistry Read
+```
+cargo run -p examples-avsregistry-read
```
Comment (dralves) [inline examples/README.md:12]: can you add some test as to what these do?

File: examples/README.md [hunk 4]
```diff
@@ -3,7 +3,19 @@
...
+```
+
+## Anvil utils(This requires a local anvil instance running using docker )
```
Comment (dralves) [inline examples/README.md:15]: can you add some test as to what these do?

File: testing/testing-utils/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+### Anvil Utilities
+
+#### Anvil Constants
+Provides utilities for interacting with local Anvil instances. Key components include:
+
+- CONTRACTS_REGISTRY: The address of the local Anvil Contracts Registry.
+- ANVIL_RPC_URL: A static instance of the local Anvil RPC URL configured with required fillers.
+
+#### Key Functions
+- get_service_manager_address(): Retrieves the address of the service manager contract.
```
Comment (dralves) [inline testing/testing-utils/README.md:16]: nice!

File: examples/README.md (hunk 3 above)
Comment (supernovahs) [reply examples/README.md:12]: Made a push , the command was wrong . wdym by test?

File: crates/services/bls_aggregation/README.md (hunk 2 above)
Comment (supernovahs) [reply crates/services/bls_aggregation/README.md:3]: we don't have one . leaving it as a placeholder

File: crates/services/avsregistry/README.md (hunk 1 above)
Comment (supernovahs) [reply crates/services/avsregistry/README.md:1]: we don't have one . leaving it as a placeholder

File: examples/README.md (hunk 4 above)
Comment (supernovahs) [reply examples/README.md:15]: wdym by test?

Comment (dralves) [reply examples/README.md:15]: I meant text, sorry

File: examples/README.md (hunk 3 above)
Comment (dralves) [reply examples/README.md]: same, text

**Pull request merged**